corehq/apps/app_manager/templates/app_manager/ng_partials/base_summary_view.html

```diff
@@ -0,0 +1,38 @@
+{% load i18n %}
```

Comment:
I'm a bit confused why this doesn't inherit from `style/two_column.html`? Or a least a flavor of that where you can override the sidebar content and main content with angular stuff? I don't like the idea of having to maintain the style of the two column structure in multiple places...

Are you intending to refresh the entire two-column view with this template? Is there a reason why the side bar and content all have to be refreshed at once? Maybe I'm just not understanding what's happening here...